gamemaster/members.php

```diff
@@ -596,7 +596,7 @@ function registerTurn() {
 	function registerNMRs($nmrs) {
 		global $DB;
 	
-		foreach( $this->ByStatus['Playing'] as $Member ){
+		foreach( $this->ByID as $Member ){

```

Comment:
What is this change doing?

Reply:
Instead of iterating only over active/playing members, left member are also included. So missed phases will be also entered for left members. As this data is only used for displaying in the game panel who missed the last cycle and this is only shown for active members, the change only gets relevant if somebody returns or a position is taken over. But nevertheless it seems to be more consistent to take note of missed turns by every member who could have given orders theoretically.

Reply:
Oh okay, that's a good design change!